docs/developer/products/api.mdx

```diff
@@ -776,3 +776,243 @@ Here's a more complex GraphQL query that combines all of the above information:
   }
 }
 ```
+
+## Create Products
```

Comment:
What do you think about mention this section in configuation: https://saleor-docs-git-cxe-885-create-product-api-guide-saleorcommerce.vercel.app/developer/products/configuration#creating-an-example-product
There is a description how to configure a product, I think we can link this API creation guide there.